lib/vcr/cassette/serializers/psych.rb

```diff
@@ -28,7 +28,9 @@ def file_extension
         # @return [String] the YAML string
         def serialize(hash)
           handle_encoding_errors do
-            ::Psych.dump(hash)
+            result = ::Psych.dump(hash)
+            result.gsub!(": \n", ": null\n") # set canonical null value in order to avoid trailing whitespaces
+            result
```

Comment:
```suggestion
            ::Psych.dump(hash).gsub(": \n", ": null\n") # set canonical null value in order to avoid trailing whitespaces
```

Reply:
My guess: the in-place substitution, which doesn't create new string value, is cheaper. Do we care, here?